src/applications/check-in/day-of/tests/e2e/see-staff-display/SeeStaff.routing.cypress.spec.js

```diff
@@ -40,29 +41,13 @@ describe('Check In Experience', () => {
     it('Navigation routing hitting no on every page', () => {
       cy.injectAxeThenAxeCheck();
       Demographics.attemptToGoToNextPage('no');
-
-      SeeStaff.validatePageLoaded();
       SeeStaff.selectBackButton();
-
-      Demographics.validatePageLoaded();
       Demographics.attemptToGoToNextPage();
-
-      EmergencyContact.validatePageLoaded();
       EmergencyContact.attemptToGoToNextPage('no');
-
-      SeeStaff.validatePageLoaded();
       SeeStaff.selectBackButton();
-
-      EmergencyContact.validatePageLoaded();
       EmergencyContact.attemptToGoToNextPage();
-
-      NextOfKin.validatePage.dayOf();
       NextOfKin.attemptToGoToNextPage('no');
-
-      SeeStaff.validatePageLoaded();
       SeeStaff.selectBackButton();
-
-      NextOfKin.validatePage.dayOf();
```

Comment:
why remove all the page load validations?

Reply:
I added these back in. When i was looking at it the first time they seemed unnecessary at the time but I see why it's important now. Added the validations back.